team-telnyx/webrtc
# feat(WEBRTC-141): added new fields to show friendly messages of errors

- added new fields to show friendly messages of errors

## 📝 To Do


## ✋ Manual testing

1. 'npm i' && 'npm run build'
2. navigate into `examples/react`
3. 'npm i' && 'npm setup' && 'npm start'
4. Make a call to an invalid number and see on logs with have theses fields
```
sipCallId: "70555126-56dd-4647-be2e-2a5e0f87677b"
sipCode: 404
sipReason: "Invalid number: 23423423423423423423"
```


## 🦊 Browser testing

### Desktop


## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |![image](https://user-images.githubusercontent.com/16343871/89942251-9bc26b80-dbf2-11ea-9381-a11c2c930bca.png)|
